src/node_api.cc

```diff
@@ -570,8 +570,7 @@ class SetterCallbackWrapper
 
   /*virtual*/
   void SetReturnValue(napi_value value) override {
-    node::FatalError("napi_set_return_value",
-      "Cannot return a value from a setter callback.");
+    assert(false);  // Cannot return a value from a setter callback
```

Comment:
Yea, that’s why I said it won’t work on Windows. ;) On POSIX it will work right now but it’s most definitely not something we should do, copying all the relevant code from node.cc would be a better option than using internal symbols from the node binary.